x-pack/plugins/observability/docs/openapi/slo/paths/s@{spaceid}@api@slos@_delete_instances.yaml

```diff
@@ -0,0 +1,40 @@
+post:
+  summary: Batch delete rollup and summary data for the matching list of sloId and instanceId
```

Comment:
nit: This gets repeated in 3 places. Maybe we should reuse from a const? I noticed it's not only the summary part that gets repeated, but the whole post part.

Reply:
Ok makes sense